packages/sync-actions/test/product-sync-base.spec.js

```diff
@@ -837,4 +837,28 @@ describe('Actions', () => {
       { action: 'setProductAttribute', name: 'indexedAttr', value: 'newValue' },
     ])
   })
+
+  test('should not fail when type is not passed', () => {
+    const before = {
+      attributes: [
+        {
+          type: 'text',
+          name: 'batteryIncluded',
+          value: 'No',
+        },
+      ],
+    }
+
+    const now = {
+      attributes: [
+        {
+          name: 'batteryIncluded',
+          value: 'No',
+        },
+      ],
+    }
+    const actions = productsSync.buildActions(now, before)
+
+    expect(actions).toEqual([])
+  })
```

Comment:
that test seems to be invalid since product attributes doesnt contain `type`.
could we replace that with the failed case regarding money attribute.

Reply:
@markus-azer just updated the test 😄  -> https://github.com/commercetools/nodejs/pull/1975/files#diff-253d86a8300336f9ac631efa60afa645a80986174ae0fe3ea42ebfbecf7d369eR841-R869